lib/diagnostic.js

```diff
@@ -6,14 +6,16 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+let upper = ''
+const normalizeWords = function (words) {
+  let upper = words.toUpperCase();
+  return upper
```

Comment:
Make sure you are splitting up the string into an array first, using something like `words.split(' ')`